zeppelin-jupyter/src/main/java/org/apache/zeppelin/jupyter/nbformat/Nbformat.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.jupyter.nbformat;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+
+/**
+ *
+ */
+public class Nbformat {
+
+  @SerializedName("metadata")
+  private Metadata metadata;
+
+  @SerializedName("nbformat")
+  private int nbformat;
+
+  @SerializedName("nbformat_minor")
+  private int nbformatMinor;
+
+  @SerializedName("cells")
+  private List<Cell> cells;
+
+  public Metadata getMetadata() {
+    return metadata;
+  }
+
+  public int getNbformat() {
```

Comment:
The reason why I talked about the version is I implemented it from latest version. I've found some changes from previous version but not sure if that's all or not. I'll try to add some object to support previous version